images/macos/software-report/SoftwareReport.Common.psm1

```diff
@@ -24,11 +24,32 @@ function Get-RVersion {
     $rVersion = Run-Command "R --version | grep 'R version'" | Take-Part -Part 2
     return "R $rVersion"
 }
+
 function Get-RustVersion {
     $rustVersion = Run-Command "rustc --version" | Take-Part -Part 1
     return "Rust $rustVersion"
 }
 
+function Get-RustfmtVersion {
+    $version = Run-Command "rustfmt --version" | Take-Part -Part 1
+    return "Rustfmt $version"
+}
+
+function Get-RustdocVersion {
+    $version = Run-Command "rustdoc --version" | Take-Part -Part 1
+    return "Rustdoc $version"
+}
+
+function Get-CargoVersion {
```

Comment:
nitpick: I suggest rename
- `Get-CargoVersion` to `Get-RustCargoVersion`
- `Get-ClippyVersion` to `Get-RustClippyVersion`
So all Rust related things would have prefix "Rust"